File: service-car-rental-java/src/main/java/com/acme/services/carRental/java/Worker.java
package com.acme.services.carRental.java;

import io.infinitic.workers.InfiniticWorker;

public class Worker {
    public static void main(String[] args) {
        // infinitic.yml resource is from the dependant "contracts" module
        try (InfiniticWorker worker = InfiniticWorker.fromConfigResource("/infinitic.yml", "/carRental.yml")) {
            worker.start();
        }
    }
}

File: service-car-rental-kotlin/src/main/kotlin/com/acme/services/carRental/kotlin/Worker.kt
package com.acme.services.carRental.kotlin

import io.infinitic.workers.InfiniticWorker

fun main() {
    // infinitic.yml resource is from the dependant "contracts" module
    InfiniticWorker.fromConfigResource("/infinitic.yml", "/carRental.yml").use { worker ->
        worker.start()
    }
}
